sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -64,6 +64,28 @@ Objects and arrays [are made reactive](/#H4sIAAAAAAAAE42QwWrDMBBEf2URhUhUNEl7c21
 
 In non-runes mode, a `let` declaration is treated as reactive state if it is updated at some point. Unlike `$state(...)`, which works anywhere in your app, `let` only behaves this way at the top level of a component.
 
+## `$state.frozen`
+
+Similar to `$state`, `$state.frozen` is also declared and can be used in many of the same ways (including on classes). However, the properties of any object and arrays are treated as read-only, and cannot be mutated. So if you intend to use objects as state and you want to mutate their properties and have reactivity work by default, it's recommended you use `$state` instead.
```

Comment:
The two portions of the intro sentence are a bit inconsistent with each other. I'd say either "is also declared and used" or "can also be declared and used"

In the second sentence it somewhat sounds like you might be referring to properties of arrays, so I think it could be reworded as "object properties and arrays" or "arrays and the properties of any object"

The third sentence has three conditional clauses joined by "and" and so I think could be simplified. I also think "work by default" raises some questions of if it doesn't work by default whether you can make some change to get non-default behavior where it does work. I might reword to something like "If you want object state that has reactivity, it's recommended to use `$state` instead."